include/openmc/physics.h

```diff
@@ -87,6 +87,10 @@ void sample_fission_neutron(int i_nuclide, const Reaction& rx, double E_in,
 void inelastic_scatter(const Nuclide& nuc, const Reaction& rx, Particle& p);
 
 void sample_secondary_photons(Particle& p, int i_nuclide);
+  
+//！Split or Roulette particles based their weight and the lower weight window bound.
```

Comment:
It appears you've added some weird unicode explanation mark. Would be best to replace it with good ol' ascii